### PR TITLE
feat: Add support for the Learner Record MFE

### PR DESCRIPTION
**Pull request by arbrandes**
This adds support for the Learner Record MFE.
    
And while we're at it:

  * Fix development mode
  * Fix the OAUTH2 LMS configuration variables; it wasn't possible to change them in production

### CODE REVIEW

**Comment by arbrandes** [comment]
@regisb, FYI.

**Comment by Faraz32123** [comment]
Hi @arbrandes, thanks for creating this PR.
Can you please add some comments here about the context of this PR. It would be really great.

**Comment by arbrandes** [comment]
Hi @Faraz32123!   The main purpose of this PR is to add the [Learner Record MFE](https://github.com/openedx/frontend-app-learner-record/) to Tutor.  Because this MFE is intended to deprecate the legacy learner-facing records UI, in a conversation with @regisb we agreed that adding it to `tutor-credentials` makes more sense than to `tutor-mfe`, for the same reason the ecommerce MFEs are a part of `tutor-ecommerce`.

Also, in testing this I noticed that launching `tutor dev` with the credentials plugin enabled was not working, so I fixed it.

**Comment by arbrandes** [comment]
Oh, and the idea is to have it be supported in Quince, so I would expect this to be pulled from nightly into master pretty soon.

**Comment by Faraz32123** [comment]
Hi @arbrandes, this PR looks good to me. In addition, can u please add some context, use or steps to access the learner record mfe and update readme according to the new settings that can be configured. It would be good keeping the readme updated. Thanks.

**Comment by arbrandes** [comment]
@Faraz32123, no progress, and no worries.  I've been busy trying to fix other MFEs, and haven't been able to get to this one again.  I'll try and do that today, though.

**Comment by Faraz32123** [comment]
> @Faraz32123, no progress, and no worries. I've been busy trying to fix other MFEs, and haven't been able to get to this one again. I'll try and do that today, though.

Thank you so much, I'll be waiting for it!

File: tutorcredentials/plugin.py [hunk 1]
```diff
@@ -59,6 +61,38 @@
...
+            "learner-record": {
+                "repository": "https://github.com/openedx/frontend-app-learner-record",
+                "refs": get_github_refs_path("openedx/frontend-app-learner-record"),
+                "port": learner_record_port,
+            },
+        }
+    )
+    return apps
+
+tutor_hooks.Filters.ENV_TEMPLATE_VARIABLES.add_items(
```
Comment (regisb) [inline tutorcredentials/plugin.py:83]: We shouldn't have to add this variable. We can now write `get_mfe("learner-record")['port']`.

File: tutorcredentials/plugin.py [hunk 2]
```diff
@@ -59,6 +61,38 @@
...
+    [("learner_record_port", learner_record_port)]
+)
+
+tutor_hooks.Filters.IMAGES_BUILD.add_item(
```
Comment (regisb) [inline tutorcredentials/plugin.py:87]: This should be taken care of automatically by tutor-mfe, right?

Comment (arbrandes) [reply tutorcredentials/plugin.py:87]: It should, now, yes!

**Comment by arbrandes** [comment]
This should now be in a reviewable state.  Tested working in nightly.

**Review by regisb — APPROVED** [review]
LGTM!

**Pull request merged**